### PR TITLE
{geo,tools}[GCCcore/11.3.0,foss/2022a] tcsh v6.24.01, time v1.9, WPS v4.4, ...

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

Depends on PR #15962

### CODE REVIEW

File: easybuild/easyconfigs/w/WRF/WRF-4.4-foss-2022a-dmpar.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    },
+}]
+patches = [
+    'WRF-%(version)s_netCDF-Fortran_separate_path.patch',
+]
+# The problem: sources from git clone have different checksum every time
+# checksums = [
+#     '0d68df19766ad03041126c1531cb115e2d2a69043df96314fd30d897cfca9e4c',  # v4.4.tar.gz
+#     '0e37c8a7bb4d25947083bdb6d0f2a9f4fdb825c88f6cb10c59b7580fe3d129ff',  # WRF-4.4_netCDF-Fortran_separate_path.patch
+# ]
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/w/WRF/WRF-4.4-foss-2022a-dmpar.eb:33]: To make the style checker happy, I would use `None` for the checksum from the `git clone` command.
```suggestion
checksums = [
    None,  # v4.4.tar.gz
    '0e37c8a7bb4d25947083bdb6d0f2a9f4fdb825c88f6cb10c59b7580fe3d129ff',  # WRF-4.4_netCDF-Fortran_separate_path.patch
]
```

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Comment by maxim-masterov** [comment]
not sure I completely understand the reason for failure on `jsc-zen2`

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
zen2-ubuntu-2204 - Linux Ubuntu 22.04.1 LTS (Jammy Jellyfish), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.6
See https://gist.github.com/a76fd9d968042f474de4d94f7a277df0 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bwd-rockylinux-90 - Linux Rocky Linux 9.0 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.10
See https://gist.github.com/7be5136da192e17d94e0a4d09bfda47c for a full test report.

**Comment by SebastianAchilles** [comment]
> not sure I completely understand the reason for failure on `jsc-zen2`

I also do not see an obvious reason. Maybe there was a short problem with the filesystem. The second build on `jsc-zen2` worked.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @maxim-masterov!

**Comment by SebastianAchilles** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**